lnwallet/commitment.go

```diff
@@ -17,6 +17,10 @@ import (
 // anchorSize is the constant anchor output size.
 const anchorSize = btcutil.Amount(330)
 
+// DefaultAnchorsCommitMaxFeeRateSatPerVByte is the default max fee rate in
```

Comment:
Should copy over the portion of the commit body that estimates the impact of this value on the number of available HTLC slots.

Reply:
Done